### PR TITLE
Add logo and header to README

### PR DESCRIPTION
**Pull request by LucianBuzzo**

### CODE REVIEW

**Comment by fisehara** [comment]
@balena-ci I self-certify!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-# Pine.js
+<div align="center">
+  <img width="400" height="auto" src="https://raw.githubusercontent.com/balena-io/pinejs/master/pinejs.png">
```
Comment (fisehara) [inline README.md:2]:
``` suggestion
<img width="400" height="auto" src="pinejs.png">
```

**Review by fisehara — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (LucianBuzzo) [reply README.md:2]: IIRC this won't work when you view the repo README, but I may be wrong...

Comment (fisehara) [reply README.md:2]: As long as the view is here in GH it should show up well. Is this needed for landr or for local cloned repo view?

Comment (LucianBuzzo) [reply README.md:2]: @fisehara Yeh I believe so, it's the approach I've always taken and I think it's because in some circumstances this path won't work.

Comment (fisehara) [reply README.md:2]: Okay, fine for me then :)

**Review by fisehara — APPROVED** [review]

**Pull request merged**